components/monitors/cu_consolemon/src/lib.rs

```diff
@@ -585,9 +577,29 @@ impl CuMonitor for CuConsoleMon {
 
         // Start the main UI loop
         thread::spawn(move || {
+            let backend = CrosstermBackend::new(stdout());
+
+            setup_terminal();
+
+            // nukes stderr into orbit as anything in the stack can corrupt the terminal
+            let dev_null = File::open("/dev/null").expect("Could not get /dev/null");
+            let dev_null_fd = dev_null.as_raw_fd();
+            let original_stderr_fd = unsafe { dup(2) };
+            unsafe {
+                dup2(dev_null_fd, 2);
```

Comment:
Does the libc create have a constant for stderr or do you have to use `2`? If not, could be good to use a var to denote that. I had to double check https://en.wikipedia.org/wiki/File_descriptor

Reply:
ha yes, libc::STDERR_FILENO  it is